src/installation/guides/fde.md

```diff
@@ -0,0 +1,254 @@
+
+On a UEFI system, the EFI system partition also needs to be mounted.
+
+```
+# mkfs.vfat /dev/sda1
+# mkdir -p /mnt/boot/efi
+# mount /dev/sda1 /mnt/boot/efi
+```
+
+Before we enter the chroot to finish up configuration, we do the actual install.
+
+```
+# xbps-install -Sy -R http://auto.voidlinux.org/current -r /mnt base-system lvm2 cryptsetup grub
+[*] Updating `http://auto.voidlinux.org/current/x86_64-repodata' ...
+x86_64-repodata: 1516KB [avg rate: 83KB/s]
+`http://auto.voidlinux.org/current' repository has been RSA signed by "Void Linux"
+Fingerprint: 60:ae:0c:d6:f0:95:17:80:bc:93:46:7a:89:af:a3:2d
+Do you want to import this public key? [Y/n] y
+123 packages will be downloaded:
+...
+```
+
+UEFI systems will have a slightly different package selection. The installation
+command for a UEFI system will be as follows.
+
+```
+# xbps-install -Sy -R http://auto.voidlinux.org/current -r /mnt base-system cryptsetup grub-x86_64-efi lvm2
+```
+
+When it's done, we can enter the `chroot` and finish up the configuration.
+
+```
+# chroot /mnt
+# chown root:root /
+# chmod 755 /
+# passwd root
+# echo voidvm > /etc/hostname
+# echo "LANG=en_US.UTF-8" > /etc/locale.conf
+# echo "en_US.UTF-8 UTF-8" >> /etc/default/libc-locales
+# xbps-reconfigure -f glibc-locales
+```
+
+The next step is editing `/etc/fstab`, which will depend on how you configured
+and named your filesystems. For this example, the file should look like this:
+
+```
+# <file system>	   <dir> <type>  <options>             <dump>  <pass>
+tmpfs             /tmp  tmpfs   defaults,nosuid,nodev 0       0
+/dev/voidvm/root  /     xfs     defaults              0       0
+/dev/voidvm/home  /home xfs     defaults              0       0
+/dev/voidvm/swap  swap  swap    defaults              0       0
+```
+
+UEFI systems will also have an entry for the EFI system partition.
+
+```
+/dev/sda1	/boot/efi	vfat	defaults	0	0
+```
+
+Next, configure `grub` to be able to unlock the filesystem. Add the following
+line to `/etc/default/grub`:
+
+```
+GRUB_ENABLE_CRYPTODISK=y
+```
+
+Next, the kernel needs to be configured to find the encrypted device. First,
+find the UUID of the device.
+
+```
+# lsblk -f
+NAME                 FSTYPE      LABEL UUID                                   MOUNTPOINT
+sda                                                                           
+└─sda1               crypto_LUKS       135f3c06-26a0-437f-a05e-287b036440a4   
+  └─voidvm        LVM2_member       gjOBNB-SXiK-qcGn-uHxp-ZdBZ-14wd-4FIL7p 
+    ├─voidvm-root xfs         root  b4e4aa35-e819-42a4-bbfc-cdcd74f1df8a   /
+    ├─voidvm-swap swap              1f8e9ca3-65bc-45b8-b7d0-6300587dfcf8   [SWAP]
+    └─voidvm-home xfs         home  b6cedb19-d8f6-4629-8519-cd2e7279ec2c   /home
+sr0                                                                           
+```
+
+Edit the `GRUB_CMDLINE_LINUX_DEFAULT=` line in `/etc/default/grub` and add
+`rd.auto=1` and `cryptdevice=UUID=<UUID>:lvm`. Make sure the UUID matches from
```

Comment:
I tried using cryptdevice and failed, perhaps because I wasn't using rd.auto. My setup is using rd.luks.uuid now, and it works well enough :)

Reply:
Is that using `rd.luks.uuid` without `cryptdevice`?